dotcom-rendering/src/palette.ts

```diff
@@ -2638,9 +2829,13 @@ const paletteColours = {
 		light: blockQuoteFillLight,
 		dark: blockQuoteFillDark,
 	},
-	'--quoted-block-quote-styles': {
-		light: quotedBlockquoteStylesLight,
-		dark: quotedBlockquoteStylesDark,
+	'--block-quote-text-styles': {
+		light: blockquoteTextStylesLight,
+		dark: blockquoteTextStylesDark,
+	},
+	'--block-quote-link-styles': {
+		light: blockQuoteLinkStylesLight,
+		dark: blockQuoteLinkStylesDark,
```

Comment:
Not a blocker: I think you can drop the `styles` suffix and make `bloquote` one word.

Reply:
Yes, I'll do this. There's actually a "rule" in editing that says "if you have to repeat sth everywhere then it probably doesn't need to be there at all"! (my words)